charts/spire/charts/spire-server/templates/configmap.yaml

```diff
@@ -35,6 +35,9 @@
 {{- if ne $tlsFederationCount 1 }}
 {{-   fail "You must enable one and only one federation TLS configuration" }}
 {{- end }}
+{{- if and (ne .Values.keyManager.awsKMS.key_identifier_file "") (ne .Values.keyManager.awsKMS.key_identifier_value "") }}
```

Comment:
Should `key_metadata_file` be added to this check? From if-else construct below it seems like it will be ignore if either of these two is set.

Reply:
@faisal-memon we just chatted on slack about switching the api to have enabled values, and follow the camel case convention.